parsers/src/main/scala/uk/gov/ons/addressIndex/crfscala/CrfParser.scala

```diff
@@ -9,12 +9,23 @@ trait CrfParser {
   val crfScala: CrfScalaJniImpl = new CrfScalaJniImpl()
 
   //TODO scaladoc
-  def tag(i: Input, features: CrfFeatures, tokenable: CrfTokenable): String = {
+  def tag(input: Input, features: CrfFeatures, tokenable: CrfTokenable): Seq[CrfTokenResult] = {
+    //todo optimise file
     val currentDirectory = new java.io.File(".").getCanonicalPath
     val modelPath = s"$currentDirectory/parsers/src/main/resources/addressCRFA.crfsuite"
```

Comment:
Still in test phase about deploying this with the file; this is for a 'working solution'